package.json

```diff
@@ -62,7 +62,10 @@
     "@types/react": "^18.2.36",
     "react-is": "^18.2.0",
     "@babel/runtime": "^7",
-    "nth-check": "^2.0.1"
+    "nth-check": "^2.0.1",
+    "@typescript-eslint/eslint-plugin": "^6.14.0",
+    "@typescript-eslint/parser": "^6.14.0",
+    "fork-ts-checker-webpack-plugin": "^6.5.3"
```

Comment:
Why are these resolutions required?

Reply:
When updating typescript to v5, CRA have those libs outdated, causing some errors 
```
@typescript-eslint/eslint-plugin and @typescript-eslint/parser: Parsing error: DeprecationError: 'originalKeywordKind' has been deprecated since v5.0.0 and can no longer be used. Use 'identifierToKeywordKind(identifier)' instead.

fork-ts-checker-webpack-plugin: TypeError: Cannot set property mark of #<Object> which has only a getter
```
Using the versions added in resolutions solve the errors